aws/v14.2.1/release.diff

```diff
@@ -0,0 +1,69 @@
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.6.1                                                     version: 1.6.1
+  - componentVersion: 1.3.1                                          - componentVersion: 1.3.1
+    name: cert-manager                                                 name: cert-manager
+    version: 2.7.1                                                     version: 2.7.1
+  - name: chart-operator                                             - name: chart-operator
+    version: 2.14.0                                                    version: 2.14.0
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.19.3                                                    version: 1.19.3
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.4.1                                                     version: 1.4.1
+  - componentVersion: 0.7.6                                          - componentVersion: 0.7.6
+    name: external-dns                                                 name: external-dns
+    version: 2.3.1                                                     version: 2.3.1
+  - componentVersion: 3.6.0                                          - componentVersion: 3.6.0
+    name: kiam                                                         name: kiam
+    version: 1.7.1                                                     version: 1.7.1
+  - componentVersion: 1.9.7                                          - componentVersion: 1.9.7
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.3.1                                                     version: 1.3.1
+  - componentVersion: 0.4.1                                          - componentVersion: 0.4.1
+    name: metrics-server                                               name: metrics-server
+    version: 1.3.0                                                     version: 1.3.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.10.1                                                    version: 1.10.1
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.7.2                                                     version: 1.7.2
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    releaseOperatorDeploy: false                                <
```

Comment:
@rossf7 can you have a look if this is correct? I just only made a copy via devctl from former release `14.2.0`

Reply:
@njuettner Yes this is correct. With cluster-operator 4.7.0 we create an app-operator per WC and its no longer managed by release-operator.

https://github.com/giantswarm/cluster-operator/blob/master/CHANGELOG.md#370---2021-03-15